tests/test_otogit_sync_gate.py

```
@@ -0,0 +1,36 @@
+"""טוקן שלא הוגדר מעולם אינו טוקן שנשבר.
+
+הסנכרון ל-otogit האדים על כל מיזוג ל-main מרגע שנוצר, כי
+‏OTOGIT_TOKEN טרם הוגדר. אדום קבוע מפסיק להיות אות — וזה מסוכן
+יותר מהיעדר סנכרון, כי הוא מרגיל את העין להתעלם מאדום.
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)